packages/flutter/test/widgets/sliver_cross_axis_group_test.dart

```diff
@@ -562,9 +617,27 @@ Widget _buildSliverCrossAxisGroup({
           scrollDirection: scrollDirection,
           reverse: reverse,
           controller: controller,
-          slivers: <Widget>[SliverCrossAxisGroup(slivers: slivers)],
+          slivers: <Widget>[SliverCrossAxisGroup(slivers: slivers), ...otherSlivers],
         ),
       ),
     ),
   );
 }
+
+class TestDelegate extends SliverPersistentHeaderDelegate {
+  TestDelegate();
+
+  @override
+  double get maxExtent => 60.0;
```

Comment:
Can you add tests where the delegate maxExtent != minExtent?

Reply:
For each case listed above. :)
In the case of SliverAppBar, the expandedHeight is what creates the same effect as minExtent < maxExtent.

Reply:
Sure, I added some, and did it for SliverAppBar as well.